core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataMap.java

```diff
@@ -654,6 +656,7 @@ public boolean isScanRequired(FilterResolverIntf filterExp) {
         startIndex++;
       }
     }
+    ExplainCollector.setTotalBlocklets(numBlocklets);
```

Comment:
This is just for one datamap blocklets, where are you summing all bloclklets?

Reply:
ok. I will change setTotalBlocklets to sum it